src/JsonHelper.cc

```diff
@@ -102,6 +109,9 @@ void JsonHelper::_saveGeoCoordinate(const QGeoCoordinate&   coordinate,
     if (writeAltitude) {
         coordinateArray << coordinate.altitude();
     }
+    if (type != NULL) {
```

Comment:
Calling `_saveGeoCoordinate` with writeAltitude=false and type!=NULL will write 3 entries, where the type can be misunderstood as altitude when reading. This is not an issue in the current implementation, but can become an issue when implementing future changes or pulling features from upstream. 
Consider placing a comment here so that we remember it when reviewing future changes